.github/actions/setup-gradle-build/action.yml

```diff
@@ -0,0 +1,33 @@
+name: Setup Gradle
+description: Sets up the environment to run Gradle
+
+inputs:
+  gradle-jvm-args:
+    description: "JVM args to pass to Gradle"
+    required: true
+    default: "-Xmx1g"
```

Comment:
Please add this explanation as comment in the code. It provides the reasoning for this setting.

Reply:
Comment added ✅

I haven't got used to write code comments yet 😅 Personally, in general, I don't write code comments since they get outdated easily. 
When debugging stuff I usually look at the corresponding Pull Request to see the discussion with other people, how the code evolved and what other alternative approaches were considered. This IDE feature is extremely helpful:
<img width="332" alt="image" src="https://github.com/pinterest/ktlint/assets/36954793/b2f08b4f-98e1-4d93-8285-d092d266a209">
I'll try to bear in mind to comment out things I normally would put as an explanation comment in a PR :)

Reply:
I agree with:
> I haven't got used to write code comments yet 😅 Personally, in general, I don't write code comments since they get outdated easily.

But comments which explain the reason why something is done are less vulnerable to that opposed to comments that try to explain what the code does.